app/redux/reducers/CollapsedSettingsReducer.js

```diff
@@ -224,15 +212,53 @@ const CollapsedSettingsReducer = (state = defaultState, action) => {
     }
 };
 
+const createDefaultState = () => {
+    const { height, width } = Dimensions.get('window');
+
+    if (width > 350) {
+        return {
+            metric1: AVG_VELOCITY_METRIC,
+            quantifier1: LAST_REP_QUANTIFIER,
+            metric2: RPE_METRIC,
+            quantifier2: EMPTY_QUANTIFIER,
+            metric3: ROM_METRIC,
+            quantifier3: MIN_QUANTIFIER,
+            metric4: DURATION_METRIC,
+            quantifier4: MIN_QUANTIFIER,
+            metric5: RPE1RM_METRIC,
+            quantifier5: EMPTY_QUANTIFIER,
+            currentCollapsedMetricRank: null,
+            isEditingMetric: false,
+            isEditingQuantifier: false,
+        };
+    } else {
+        return {
+            metric1: AVG_VELOCITY_METRIC,
+            quantifier1: LAST_REP_QUANTIFIER,
+            metric2: RPE_METRIC,
+            quantifier2: EMPTY_QUANTIFIER,
+            metric3: ROM_METRIC,
+            quantifier3: MIN_QUANTIFIER,
+            metric4: RPE1RM_METRIC,
+            quantifier4: EMPTY_QUANTIFIER,
+            metric5: DURATION_METRIC,
+            quantifier5: MIN_QUANTIFIER,
+            currentCollapsedMetricRank: null,
+            isEditingMetric: false,
+            isEditingQuantifier: false,
+        };
+    }
+}
```

Comment:
- [ ] Again, this only DEFAULTS NEW APPS to it. We want to OVERRIDE existing metrics with it

Reply:
This will be a new PR.